src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerGene.tsx

```diff
@@ -31,7 +33,9 @@ const DrawerGene: FunctionComponent<DrawerGeneProps> = (
 
       <dd className="clearfix">
         <label htmlFor="">Description</label>
-        <div className={styles.details}>{ensObjectInfo.description}</div>
+        <div className={styles.details}>
+          {ensObjectInfo.description || '--'}
```

Comment:
Do we actually use double hyphens (`--`) instead of em-dash (`—`)?

Reply:
Yes, Andrea preferred it to be '--'.

Reply:
👌